[bitnami/wavefront-hpa-adapter] Add deprecation notice (#17200)
bitnami/charts: bitnami/wavefront-hpa-adapter/templates/NOTES.txt

> This Helm chart is deprecated
> 
> The upstream project has been discontinued and no new features.

Reviewer: ```suggestion
This Helm chart is deprecated on our side and will not receive new updates.
```
